target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -42,10 +42,12 @@ ramips_setup_interfaces()
 	mikrotik,routerboard-750gr3)
 		ucidef_set_interfaces_lan_wan "lan2 lan3 lan4 lan5" "wan"
 		;;
-	ubiquiti,edgerouterx|\
-	ubiquiti,edgerouterx-sfp)
+	ubiquiti,edgerouterx)
 		ucidef_set_interface_lan "eth0 eth1 eth2 eth3 eth4"
 		;;
+	ubiquiti,edgerouterx-sfp)
+		ucidef_set_interfaces_lan_wan "eth1 eth2 eth3 eth4 eth5" "eth0"
+		;;
 	*)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
```

Comment:
Should maybe this line be changed to:
"lan1 lan2 lan3 lan4 SFP" "wan"
or
"lan1 lan2 lan3 lan4 lan5" "wan"

Reply:
Or maybe "eth1 eth2 eth3 eth4 eth5/SFP" "eth0" like is printed on the device

Reply:
but then seperately for the erx and erx-sfp